build.gradle

```diff
@@ -26,7 +26,7 @@ subprojects {
 
     dependencies {
         //TODO use the + sign for latest version when we start to use repository manager
-        compile 'com.kaltura:KalturaClientLib:3.3.1'
```

Comment:
Please update the version also in the API: https://github.com/kaltura/server/blob/Kajam-11.12.0/api_v3/VERSION.php#L2